### PR TITLE
[dev-v5] Add `Aside` panel in the Demo website

### PR DESCRIPTION
**Pull request by dvoituron**
# [dev-v5] Add `Aside` panel in the Demo website

Extract the HTML Headers to have data to include in the `Aside` Panel (right panel)

Aside page content can be customized using the `DemoAsidePanel.razor` file. For example, to display different content on the home page and other component pages.

```razor
@switch (Page)
{
    // Home page
    case "":
        <MarkdownViewer Route="file:HomeAside" />
        break;

    // All documentation pages
    default:
        if (Headers != null && Headers.Any(i => i.Level > 1))
        {
            <div class="demo-aside">
                <h2>On this page</h2>
                @GetHtmlHeaders()
            </div>
        }
        break;
}

```

## Component Doc

{DAFE77CA-A702-4B20-8E37-87B6B9BE9D1D}

## Home Page

{3771C43A-8FC0-4B42-B64C-A0F0CBEAF081}

### CODE REVIEW

File: examples/Demo/FluentUI.Demo.Client/Layout/DemoAsidePanel.razor.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+public partial class DemoAsidePanel
+{
+    /// <summary />
+    [Inject]
+    public NavigationManager Navigation { get; set; } = default!;
+
+    /// <summary />
+    [Inject]
+    public Demo.DocViewer.Services.DocViewerService DocViewerService { get; set; } = default!;
```
Comment (vnbaaij) [inline examples/Demo/FluentUI.Demo.Client/Layout/DemoAsidePanel.razor.cs:19]: Any particular reason for not creating an `@using`?

File: tests/Core/_StartCodeCoverage.cmd [hunk 2]
```diff
@@ -19,5 +19,5 @@ echo on
 cls
 
 dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura
-reportgenerator "-reports:coverage.cobertura.xml" "-targetdir:C:\Temp\FluentUI\Coverage" -reporttypes:HtmlInline_AzurePipelines -classfilters:"-Microsoft.FluentUI.AspNetCore.Components.DesignTokens.*" -filefilters:"-*RegexGenerator.g.cs" riskHotspotsAnalysisThresholds:metricThresholdForCrapScore=30 riskHotspotsAnalysisThresholds:metricThresholdForCyclomaticComplexity=30 minimumCoverageThresholds:lineCoverage=98
+reportgenerator "-reports:coverage.net8.0.cobertura.xml" "-targetdir:C:\Temp\FluentUI\Coverage" -reporttypes:HtmlInline_AzurePipelines -classfilters:"-Microsoft.FluentUI.AspNetCore.Components.DesignTokens.*" -filefilters:"-*RegexGenerator.g.cs" riskHotspotsAnalysisThresholds:metricThresholdForCrapScore=30 riskHotspotsAnalysisThresholds:metricThresholdForCyclomaticComplexity=30 minimumCoverageThresholds:lineCoverage=98
```
Comment (vnbaaij) [inline tests/Core/_StartCodeCoverage.cmd:22]: Need to change this again because of the merged test PR

**Review by vnbaaij — CHANGES_REQUESTED** [review]

File: examples/Demo/FluentUI.Demo.Client/Layout/DemoAsidePanel.razor.cs (hunk 1 above)
Comment (dvoituron) [reply examples/Demo/FluentUI.Demo.Client/Layout/DemoAsidePanel.razor.cs:19]: Nop... Will fixed

File: tests/Core/_StartCodeCoverage.cmd (hunk 2 above)
Comment (dvoituron) [reply tests/Core/_StartCodeCoverage.cmd:22]: Good catch

**Review by vnbaaij — APPROVED** [review]

**Pull request merged**